x-pack/legacy/plugins/uptime/server/lib/adapters/monitors/__tests__/elasticsearch_monitors_adapter.test.ts

```diff
@@ -102,7 +102,7 @@ describe('ElasticsearchMonitorsAdapter', () => {
       10
     );
     expect(fixedInterval).not.toBeNaN();
-    expect(fixedInterval).toBeCloseTo(36000, 3);
+    expect(Math.abs(36000 - fixedInterval)).toBeLessThan(100);
```

Comment:
Can we explain the constant of `36000`? It could use a comment

Reply:
I think the assertions and failures would be clearer if we wrote this as a helper function with a custom error being thrown:

```ts
const assertCloseTo = (actual: number, expected: number, precision: number) => {
  if (Math.abs(expected - actual) > precision) {
    throw new Error(`expected [${actual}] to be within ${precision} of ${actual}`)
  }
}
```